projects/go-lib/src/lib/components/go-table/go-table-utils.ts

```diff
@@ -1,11 +1,19 @@
 export function extractFieldData(key: string, obj: object): any {
   if (key) {
-    return key.split('.').reduce((p: any, c: any) => p && p[c], obj) || '';
+    return key.split('.').reduce((p: any, c: any) => {
+      let val: any = p && p[c];
+
+      if (typeof val === 'boolean') {
+        val = val.toString();
+      }
+
+      return val;
+    }, obj) || '';
```

Comment:
This logic still prevents a numeric `0` from showing since `0 || ''` will resolve to `''`. 

You can test this in the docs by removing the `+1` from https://github.com/mobi/goponents/blob/bug_table_false_values/projects/go-style-guide/src/app/features/ui-kit/components/table-docs/table-docs.service.ts#L21 so a `0` id will generate. 

Is this meant to also be fixed by this PR or will it continue to be an open issue?

Reply:
@patrick-lewandowski I thought there was something else, but I couldn't remember what it was and we didn't have an actual issue for it. I can look into that..

Honestly, maybe we should just `.toString` everything? If you need extra formatting that's what the templates are for anyway.

Reply:
The other option is take out the `|| ''` part and have everything return as expected. If it's meant to be `null` or `undefined`, then so be it. I personally don't think we should be assuming everything should default to an empty string. You can try `toString` just not sure if there would be any edge cases around it.

Reply:
That's a good point. The issue we ran into in the past when we tried to remove the `|| ''` was that it broke client side sorting because any of the columns that contained `null` or `undefined` values weren't able to be sorted.

Taking that into account, I removed the `|| ''` from this method and updated the `sortBy` method to account for `null` or `undefined` values and default them to `''` strictly inside of the `sortBy` method.

This way we're not modifying any data, correcting both bugs, and satisfying all the cases (I think).